sdmetrics/reports/single_table/plot_utils.py

```diff
@@ -89,6 +89,9 @@ def _get_similarity_correlation_matrix(score_breakdowns, columns):
     Returns:
         pandas.DataFrame
     """
+    if isinstance(columns, set):
+        columns = list(columns)
```

Comment:
Minor, but not sure what `columns` can be:
You can always cast to `list`  and this will be one dimensional:
```python
In [1]: columns_list = ['a', 'b', 'c']
In [2]: columns_set = {'a', 'b', 'c'}
In [3]: columns_tuple = ('a', 'b', 'c')

In [4]: list(columns_list)
Out[4]: ['a', 'b', 'c']

In [5]: list(columns_set)
Out[5]: ['b', 'c', 'a']

In [6]: list(columns_tuple)
Out[6]: ['a', 'b', 'c']
```
If `columns` can be a `str` then don't do this as it will split it in letters

Reply:
It should always be a list or a set.